src/sample/courseRegController.java

```diff
@@ -286,5 +297,9 @@ public void initialize(URL location, ResourceBundle resources) {
     displaycourselist();
     displaycourseregistered();
     coursesuggestion();
+    courseTitle.setText("CAT 201 Software Integrated Development Workshop kdfkslfsfhjshfjdkahahahhahahahahahahahahaha");
+    time.setText("Tuesday 3pm - 4pm ahahhahahahahahahahahahadjfhjhjsfhjksffksdhfkshdfjkshfdjdkshfljsakhsah");
+    desc.setText(
+        "The course serves to dkksjfasjkfdsfkjslkdjfaskljfksjfahahhahahahahahahahahahadjfksdhfjksadhfljashfjkshd");
```

Comment:
I put very long words so that you can see whether the words will automatically go to the next line when there are too many characters